app/Http/Kernel.php

```diff
@@ -23,14 +28,6 @@ class Kernel extends HttpKernel
      * @var array
      */
     protected $middlewareGroups = [
-        'web' => [
```

Comment:
What's the reasoning for moving these into the global middleware stack? I think it makes sense to only apply these on web routes (since API routes don't need sessions, cookies, or CSRF tokens).